app/src/main/res/layout/route_description.xml

```diff
@@ -91,13 +91,33 @@
         android:textSize="18dp"
         android:textStyle="bold"/>
 
-    <TextView
-        android:id="@+id/myAscentsTextView"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center"
-        android:layout_margin="10dp"
-        android:text="@string/my_ascends"/>
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <TextView
+            android:id="@+id/myAscentsTextView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_margin="11dp"
+            android:text="@string/my_ascends"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"/>
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="16dp"
+            android:onClick="enterAscend"
+            android:src="@android:drawable/ic_input_add"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"/>
```

Comment:
```suggestion
            app:layout_constraintBottom_toBottomOf="parent"
            style="@style/floatingActionButtonStyle"/>
```
Add a style to file `style.xml`:
```xml
<style name="floatingActionButtonStyle">
    <item name="tint">@color/colorOnPrimary</item>
    <item name="backgroundTint">@color/colorPrimary</item>
</style>
```
I'm not fully happy with the style but for now it is OK.